docs/eventing/samples/helloworld/helloworld-go/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Use the offical Golang image to create a build artifact.
 # This is based on Debian and sets the GOPATH to /go.
 # https://hub.docker.com/_/golang
-FROM golang:1.12 as builder
+FROM golang:1.14 as builder
```

Comment:
I think we are on 1.13 ?

Reply:
Yes. This should be 1.13. @haf could you please change it and send an update. I'll LGTM.

Reply:
can we get rid off go dep as well ?